doc/sphinx/source/libCEEDapi.rst

```diff
@@ -50,11 +50,11 @@ versions of :math:`\mathbf{P}`, :math:`\mathbf{G}` and :math:`\mathbf{B}`.
    Operator Decomposition
 
 Note that in the case of adaptive mesh refinement (AMR), the restrictions
-:math:`\mathbf{P}` and :math:`\mathbf{G}` will involve not just extracting sub-vectors,
+:math:`\bm{P}` and :math:`\bm{G}` will involve not just extracting sub-vectors,
 but evaluating values at constrained degrees of freedom through the AMR interpolation.
-There can also be several levels of subdomains (:math:`\mathbf{P1}`, :math:`\mathbf{P2}`,
-etc.), and it may be convenient to split :math:`\mathbf{D}` as the product of several
-operators (:math:`\mathbf{D1}`, :math:`\mathbf{D2}`, etc.).
+There can also be several levels of subdomains (:math:`\bm{P1}`, :math:`\bm{P2}`,
+etc.), and it may be convenient to split :math:`\bm{D}` as the product of several
+operators (:math:`\bm{D1}`, :math:`\bm{D2}`, etc.).
```

Comment:
```suggestion
There can also be several levels of subdomains (:math:`\bm P_1`, :math:`\bm P_2`,
etc.), and it may be convenient to split :math:`\bm{D}` as the product of several
operators (:math:`\bm D_1`, :math:`\bm D_2`, etc.).
```